src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        int index = new Random().nextInt(Color.values().length);
+        Color color = Color.values()[index];
+        return color.toString();
     }
```

Comment:
Don't create several instances of the same class if you can use only one instance for your purposes. In this case, a new `Random` object is created every time the `getRandomColor` method is called, which is unnecessary. Instead, you should create a single `Random` instance as a class field and reuse it.